### PR TITLE
Release notes 5.0.0

### PR DESCRIPTION
**Pull request by achamayou**
Aggregate release notes from dev releases and release candidates into a single 5.0.0 block.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,154 @@ All notable changes to this project will be documented in this file.
...
+
+[5.0.0]: https://github.com/microsoft/CCF/releases/tag/ccf-5.0.0
+
+In order to upgrade an existing 4.x service to 5.x, CCF must be on the latest 4.x version (at least 4.0.19).
+For more information, see [our documentation](https://microsoft.github.io/CCF/main/operations/code_upgrade.html)
+
+### Important
+
+- 5.0.0 is the last long term support release for Intel SGX. Live code upgrades to AMD SEV-SNP on [Confidential Containers on Azure Container Instances](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-confidential-overview) and [Confidential Containers Azure Kubernetes Service](https://learn.microsoft.com/en-us/azure/aks/confidential-containers-overview) are supported.
+- Request forwarding between nodes is deprecated in favour of HTTP redirects, and will be removed in 6.0.0.
```
Comment (maxtropets) [inline CHANGELOG.md:18]: Request`ing` probably?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,154 @@ All notable changes to this project will be documented in this file.
...
+  - `::siphash` is now `ccf::siphash`
+  - `::threading` is now `ccf::threading`, and `ccf/ds/thread_ids.h` has moved to `ccf/threading/thread_ids.h`
+  - `::consensus` is now `ccf::consensus`
+  - `::tls` is now `ccf::tls`
+  - `::http` is now `ccf::http`
+  - `::nonstd` is now `ccf::nonstd`
+  - `::crypto` is now `ccf::crypto`
+  - `::kv` is now `ccf::kv`
+  - `::logger` is now `ccf::logger`
+  - `::ccfapp` is now `::ccf`
```
Comment (maxtropets) [inline CHANGELOG.md:53]: Is it `::ccf` or `ccf`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,154 @@ All notable changes to this project will be documented in this file.
...
+- New endpoints `GET /gov/service/javascript-modules` and `GET /gov/service/javascript-modules/{moduleName}` to retrieve the raw JS code of the currently installed app. Note that the `{moduleName}` path parameter will need to be URL-encoded to escape any `/` characters (eg - `/foo/bar.js` should become `%2Ffoo%2Fbar.js`).
+
+---
+
+### Operations
+
+#### Configuration
+
+- The `cchost` configuration file now includes an `idle_connection_timeout` option. This controls how long the node will keep idle connections (for user TLS sessions) before automatically closing them. This may be set to `null` to restore the previous behaviour, where idle connections are never closed. By default connections will be closed after 60s of idle time.
+- A soft limit can now be set for the historical store cache in the node configuration: [`historical_cache_soft_limit`](https://microsoft.github.io/CCF/main/operations/generated_config.html#historical-cache-soft-limit).
```
Comment (maxtropets) [inline CHANGELOG.md:99]: soft `size` limit

**Review by maxtropets — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (achamayou) [reply CHANGELOG.md:18]: It's the forwarding of requests that is deprecated, will re-phrase to make this clearer.

File: CHANGELOG.md (hunk 2 above)
Comment (achamayou) [reply CHANGELOG.md:53]: It's `::ccf`.

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,155 @@ All notable changes to this project will be documented in this file.
...
+  - `::ds` is now `ccf::ds`
+  - `::siphash` is now `ccf::siphash`
+  - `::threading` is now `ccf::threading`, and `ccf/ds/thread_ids.h` has moved to `ccf/threading/thread_ids.h`
+  - `::consensus` is now `ccf::consensus`
+  - `::tls` is now `ccf::tls`
+  - `::http` is now `ccf::http`
+  - `::nonstd` is now `ccf::nonstd`
+  - `::crypto` is now `ccf::crypto`
+  - `::kv` is now `ccf::kv`
+  - `::logger` is now `ccf::logger`
```
Comment (achamayou) [inline CHANGELOG.md:52]:
```suggestion
  - `::ds` is now `::ccf::ds`
  - `::siphash` is now `::ccf::siphash`
  - `::threading` is now `::ccf::threading`, and `ccf/ds/thread_ids.h` has moved to `ccf/threading/thread_ids.h`
  - `::consensus` is now `::ccf::consensus`
  - `::tls` is now `::ccf::tls`
  - `::http` is now `::ccf::http`
  - `::nonstd` is now `::ccf::nonstd`
  - `::crypto` is now `::ccf::crypto`
  - `::kv` is now `::ccf::kv`
  - `::logger` is now `::ccf::logger`
```

**Pull request merged**